src/pages/junyoung/LoginJunyoung/LoginJunyoung.js

```diff
@@ -1,8 +1,76 @@
-import React from "react";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import "./LoginJunyoung.scss";
 
 const LoginJunyoung = () => {
-  return <div className="LoginJunyoung">LoginJunyoung</div>;
+  const navigate = useNavigate();
+
+  const goToMain = () => {
+    if (id.includes("@") && pw.length >= 5) {
+      navigate("/main-junyoung");
+    } else {
+      alert("아이디와 비밀번호를 다시 확인해주세요.");
+    }
+  };
+
+  const [id, setId] = useState("");
+  const [pw, setPw] = useState("");
+  const [active, setActive] = useState(false);
+
+  function saveUserId(event) {
+    setId(event.target.value); //이걸 모르겠음
+    // console.log("id", id);
+  }
+  const saveUserPw = event => {
+    setPw(event.target.value);
+    // console.log("pw", pw);
+  };
+
+  const activeBtn = () => {
+    return id.includes("@") && pw.length >= 5
+      ? setActive(true)
+      : setActive(false);
+  };
+
+  return (
+    <div className="wrapBox">
```

Comment:
```suggestion
    <div className="loginJunyoung">
```
scss 사용할 때 wecode 컨벤션은 컴포넌트의 이름과 동일하게 적용해야 합니다!